rerun_py/requirements-doc.txt

```diff
@@ -0,0 +1,12 @@
+griffe==0.39.0
```

Comment:
I could not get `griffe dump` to work on the SDK, after removing the `py-docs` pixi env, so I reverted back to this requirements file.

Reply:
There should be an issue for it, and it should be linked to from the pixi.toml file.